cscs-checks/libraries/gridtools/gridtools_tests.py

```diff
@@ -29,50 +35,58 @@ def __init__(self):
         self.build_system.flags_from_environ = False
         self.build_system.make_opts = ['perftests']
         self.build_system.max_concurrency = 8
-        self.postbuild_cmds = ['ls tests/regression/']
+
+    @rfm.run_before('sanity')
+    def set_sanity_patterns(self):
         self.sanity_patterns = sn.assert_found(r'perftest',
                                                self.stdout)
-        self.tags = {'scs', 'benchmark'}
-        self.maintainers = ['CB']
 
 
 @rfm.simple_test
 class GridToolsCPUBuildCheck(GridToolsBuildCheck):
```

Comment:
I would rather have a single build test parametrised over `cpu` and `gpu`.

Reply:
Done